src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1537,6 +1538,20 @@ First First():
 	}
 }
 
+
+Expression MandatoryExpression() #Expression :
```

Comment:
Sorry it should be removed, I didn't catch it.

On Monday, 20 June 2016, Tobias <notifications@github.com> wrote:

> In src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt
> <https://github.com/JSQLParser/JSqlParser/pull/283#discussion_r67623169>:
>
> > @@ -1537,6 +1538,20 @@ First First():
> >  	}
> >  }
> >
> > +
> > +Expression MandatoryExpression() #Expression :
>
> Where is this used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JSQLParser/JSqlParser/pull/283/files/401d279ef7e6ef9f6a2313bf6b5f14670d331f3d#r67623169>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEadltuFZMCPSwwA46zoxnf9VNhUCEKSks5qNaQggaJpZM4I3enb>
> .
>